src/main/java/org/mitre/synthea/engine/State.java

```diff
@@ -6,6 +6,7 @@
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
+import javafx.util.Pair;
```

Comment:
IIRC JavaFX is getting pulled out of the base JDK, so if we can avoid referencing the javafx package that would be ideal.